api/Document.json

```diff
@@ -10823,7 +10823,7 @@
               "version_added": "33"
             },
             "edge": {
-              "version_added": "18"
+              "version_added": "12"
```

Comment:
Similarly, here can you point out that the event handler attribute won't work? It would be natural to assume it does.